internal/deviations/deviations.go

```diff
@@ -481,6 +481,13 @@ func StorageComponentUnsupported(_ *ondatra.DUTDevice) bool {
 	return *storageComponentUnsupported
 }
 
+// NtpNonDefaultVrf returns true if the device does not support ntp nondefault vrf.
+// Default value is false.
+func NtpNonDefaultVrf(dut *ondatra.DUTDevice) bool {
+	logErrorIfFlagSet("deviation_ntp_non_default_vrf")
```

Comment:
since this is a new deviation we can remove the `logErrorIfFlagSet("deviation_ntp_non_default_vrf")` line. No need to define the flag, and since the flag is not present, users will not be able to set it.